testing/README.md

```diff
@@ -21,13 +21,4 @@ The compliance and testing suite generates an HTML report for every tested serve
 
 * caltech.edu server - A demo server is available at http://felcat.caltech.edu/rnaget/ serving the Pan-Cancer Analysis of Whole Genomes (PCAWG) dataset.
```

Comment:
Please remove the link to the felcat server.  There have been some hardware issues and the server is currently down with no ETA on recovery.

Reply:
Updated in b119306